src/transaction/replication.c

```diff
@@ -449,20 +448,32 @@ repl_log_insert (THREAD_ENTRY * thread_p, const OID * class_oid, const OID * ins
       LOG_REPL_RECORD *recsp = tdes->repl_records;
       int i;
 
-      for (i = 0; i < tdes->fl_mark_repl_recidx; i++)
+      if (strcmp (class_name, CT_SERIAL_NAME) != 0)
 	{
-	  if (recsp[i].must_flush == LOG_REPL_COMMIT_NEED_FLUSH && OID_EQ (&recsp[i].inst_oid, &repl_rec->inst_oid))
+	  for (i = 0; i < tdes->fl_mark_repl_recidx; i++)
```

Comment:
Do you think the begin of the loop is not needed from final index?
Just asking..

Reply:
I don't understand the meaning of comment. explain more detail please.